static/js/lib/ckeditor/plugins/ResourceLinkMarkdownSyntax.test.ts

```diff
@@ -46,4 +46,23 @@ describe("ResourceLink plugin", () => {
       '<p><a class="resource-link" data-uuid="asdfasdfasdfasdf">text here</a></p>'
     )
   })
+
+  it("should serialize multiple links to and from markdown", async () => {
+    const editor = await getEditor("")
+    markdownTest(
+      editor,
+      'dogs {{< resource_link uuid1 "woof" >}} cats {{< resource_link uuid2 "meow" >}}, cool',
+      '<p>dogs <a class="resource-link" data-uuid="uuid1">woof</a> cats <a class="resource-link" data-uuid="uuid2">meow</a>, cool</p>'
+    )
+  })
+
+  it("[BUG] does not behave well if link title ends in backslash", async () => {
```

Comment:
This test was just to document the behavior https://github.com/mitodl/ocw-studio/issues/928 🤷